src/Microsoft.Sbom.Api/SBOMValidator.cs

```diff
@@ -45,7 +57,7 @@ public async Task<bool> ValidateSbomAsync()
         return isSuccess;
     }
 
-    public async Task<bool> ValidateSbomAsync(
+    public async Task<SBOMValidationResult> ValidateSbomAsync(
```

Comment:
Unfortunately, using inheritance of SBOMValidationResult to define success and failed validation might not be a good API design pattern. The caller of ValidateSbomAsync would need to check for the type of the SBOMValidationResult object to see if the validation failed or not. Can we add the Errors property to SBOMValidationResult, add also another boolean property to define the Result of the validation and remove the abstract modifier from the class. Then we could get rid of SBOMValidationSuccess and SBOMValidationFailure classes